docs/tutorials/ethereum-local-development.md

```diff
@@ -0,0 +1,264 @@
+
+```bash
+truffle compile
+```
+
+You should find the contract's ABI at the location: `build/contracts/MetaCoin.json`, **it will be useful for indexing**.
+
+#### 2. Create a workspace
+
+Launch the Ganache tool and select the New Workspace (Ethereum option).
+
+![Create workspace](</img/ganache-create-workspace.png>)
+
+In the following window, provide a name for the workspace, link the Truffle project we just created, by clicking on **Add project** button and selecting the `truffle-config.js` file in the project's root folder. Finally, select the `Server` tab at the top.
+
+![Provide name](</img/ganache-create-workspace-1.png>)
+
+In this window, change the server configuration to the exact values reported in this image.
+
+![Server configuration](</img/ganache-create-workspace-2.png>)
+
+:::info
+**Note:** It is not mandatory to change `PORT NUMBER` or `NETWORK ID`, but they are set to the same value as Hardhat, so the rest of the Tutorial will look the same.
+
+The `AUTOMINE` option is disabled, same as in the *Hardhat* section of this tutorial, for debugging purposes. For more information, consult the [official documentation](https://trufflesuite.com/docs/ganache/reference/cli-options/#miner).
+:::
+
+#### 3. Deploy smart contract
+
+Deploy the sample smart contract by launching the following command in a console window:
+
+```bash
+truffle migrate
+```
+
+:::warning
+**Note:** Make sure the `development` configuration, in the `networks` section of `truffle-config.js` is uncommented, and has these values (which should be the same as the previous step).
+
+```
+    development: {
+     host: "127.0.0.1",     // Localhost (default: none)
+     port: 8545,            // Standard Ethereum port (default: none)
+     network_id: "1337",       // Any network (default: none)
+    },
+```
+:::
+
+Log information similar to this should be displayed:
+
+```bash
+   Deploying 'MetaCoin'
+   --------------------
+   > transaction hash:    0xcdb820827306ebad7c6905d750d07536c3db93f4ef76fd777180bdac16eaa2ca
+   > Blocks: 1            Seconds: 4
+   > contract address:    0xd095211a90268241D75919f12397b389b1062f6F
+   > block number:        329
+   > block timestamp:     1673277461
+   > account:             0x131D37F433BAf649111278c1d6E59843fFB26D28
+   > balance:             99.98855876
+   > gas used:            414494 (0x6531e)
+   > gas price:           20 gwei
+   > value sent:          0 ETH
+   > total cost:          0.00828988 ETH
+
+   > Saving artifacts
+   -------------------------------------
+   > Total cost:          0.01144124 ETH
+```
+
+Take note of the contract address, **you'll need it later**.
+
+### Archive Docker image
+
+In order to index the local Ethereum node, we need to use a pre-built Docker image of Subsquid's Ethereum Archive. You can do it, by creating a `docker-compose.archive.yml` file, and paste this:
+
+```yaml
+version: "3"
+
+services:
+  worker:
+    image: massimocodes/eth-archive-worker:latest
```

Comment:
We should use an official subsquid image here

Reply:
I know, but they are not public. I don't have access to them, or cannot create them.

Reply:
Changed